### PR TITLE
add set typeface from assets

### PR DESCRIPTION
**Pull request by YavariRaana**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,6 +4,15 @@ buildscript {
...
         jcenter()
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
+        maven{
+            url "https://dl.bintray.com/infideap2/Drawer-Behavior"
+        }
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
```
Comment (ShawnLin013) [inline build.gradle:15]: Unnecessary changes

File: build.gradle [hunk 2]
```diff
@@ -15,8 +24,11 @@ buildscript {
 
 allprojects {
     repositories {
-        google()
+        maven { url "https://maven.google.com" }
+        mavenLocal()
```
Comment (ShawnLin013) [inline build.gradle:28]: unnecessary changes

File: build.gradle [hunk 3]
```diff
@@ -15,8 +24,11 @@ buildscript {
...
         jcenter()
+        maven { url "https://jitpack.io" }
+        google()
```
Comment (ShawnLin013) [inline build.gradle:31]: unnecessary changes

**Comment by YavariRaana** [comment]
add ability to set type face from assets

**Comment by ShawnLin013** [comment]
It would be better to add a new method like `setTypefaceFromAsset` instead of modifying the existing methods.
